ATM.py

```diff
@@ -0,0 +1,120 @@
+__author__ = 'Adi Kadar Levi'
+BANK_DATA_PATH = r'C:\Users\Elior Mor Yosef\Documents\GitHub' \
+                 r'\python-training\python-training\bank.txt'
+
+
+def passwords_dictionary():
+    pass_dict = {}
+    with open(BANK_DATA_PATH, 'r') as bank_data:
+        for line in bank_data:
+            words = line.split()
+            pass_dict[words[0]] = words[1]
+    return pass_dict
+
+
+def balance_dictionary():
+    balance_dict = {}
+    with open(BANK_DATA_PATH, 'r') as bank_data:
+        for line in bank_data:
+            words = line.split()
+            balance_dict[words[0]] = words[2]
+    return balance_dict
+
+
+def check_id(id_input):
+    if id_input not in passwords_dictionary().keys():
+        print('User Not Found')
+        return False
+    else:
```

Comment:
you don't need this else, you can put the `return True` outside the if statement and it would have the same result.

Reply:
done